### PR TITLE
MovieDetail now makes real time API call

### PR DESCRIPTION
**Pull request by r4v1k**

### CODE REVIEW

File: src/components/movieDetail/MovieDetail.jsx [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
+
+function MovieDetail () {
+    const[movieDetailObj, setMovieDetailObj] = useState('');
+    var id = ImdbId();
+    useEffect(()=> {
+      getMovieDetailObj();   
+  
+    },[])
+    
+    const getMovieDetailObj= (props) => {
```
Comment (sandeepsuku) [inline src/components/movieDetail/MovieDetail.jsx]: Do we need a Props argument here ?

File: src/components/movieDetail/MovieDetail.jsx [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
+         console.log("getMovieDetail response -> " + resp.data);
+         setMovieDetailObj(resp.data);
+      })
+      .catch(err => {
+            console.error("Error " + err);
+      })
+    }     
+
+    return (<div className="movie-info">
         <div className="background-black">
```
Comment (sandeepsuku) [inline src/components/movieDetail/MovieDetail.jsx]: I feel without black background, the page looks better and consistent with landing page.

File: src/components/movieDetail/MovieDetail.jsx (hunk 1 above)
Comment (r4v1k) [reply src/components/movieDetail/MovieDetail.jsx]: no. removed

File: src/components/movieDetail/MovieDetail.jsx (hunk 2 above)
Comment (r4v1k) [reply src/components/movieDetail/MovieDetail.jsx]: removed. initially added this to look better when landing page was not ready

**Review by sandeepsuku — APPROVED** [review]

**Pull request merged**